defaults/main.yml

```diff
@@ -3,7 +3,9 @@
 # default (OpenJDK 1.7).
 # java_packages: []
 
-java_home: ""
+openjdk_version: 8
+java_home: "/usr/lib/jvm/java-8-openjdk-amd64"
```

Comment:
yes but this role is still meant to be used to install oracle-jdk when necessary, isn'it?

Reply:
In other hand it will still be possible to override `java_home`, so ok :)

Reply:
No it will just install OpenJDK. OracleJDK is still managed by the old role [peopledoc/ansible-role-oracle-java](https://github.com/peopledoc/ansible-role-oracle-java).